src/Microsoft.AspNetCore.Mvc.ViewFeatures/Internal/SaveTempDataFilter.cs

```diff
@@ -29,16 +29,6 @@ public SaveTempDataFilter(ITempDataDictionaryFactory factory)
         /// <inheritdoc />
         public void OnResourceExecuting(ResourceExecutingContext context)
         {
-        }
-
-        /// <inheritdoc />
-        public void OnResourceExecuted(ResourceExecutedContext context)
-        {
-        }
-
-        /// <inheritdoc />
-        public void OnResultExecuting(ResultExecutingContext context)
-        {
             context.HttpContext.Response.OnStarting((state) =>
```

Comment:
Should we check `ResponseStarted` here? Middleware are a thing. It would be better not to throw.